default.xml

```diff
@@ -30,15 +30,15 @@
   <project name="projects_libs.git" path="projects/projects_libs" remote="sel4devkit" revision="master" upstream="master" dest-branch="master"/>
   <project name="pruner.git" path="tools/pruner" revision="6df378e8d91856d204e4d024796b91c6802d0501" upstream="master" dest-branch="master"/>
   <project name="rumprun.git" path="tools/rumprun" revision="04464ab931c28ca1c78e807741d82a4792660183" upstream="master" dest-branch="master"/>
-  <project name="seL4.git" path="kernel" remote="sel4devkit" revision="master" upstream="master" dest-branch="master"/>
+  <project name="seL4.git" path="kernel" remote="seL4" revision="ae7798e3193a223de5c841bdf6ad319937b3403b" upstream="master" dest-branch="master"/>
   <project name="seL4_libs.git" path="projects/seL4_libs" revision="d8ae2c44e92d1bf19891344eb038752565f0b30e" upstream="master" dest-branch="master"/>
   <project name="seL4_projects_libs.git" path="projects/seL4_projects_libs" revision="b687e3d503ed685634713eae868b220bdcda8261" upstream="master" dest-branch="master"/>
-  <project name="seL4_tools.git" path="tools/seL4" remote="sel4devkit" revision="master" upstream="master" dest-branch="master">
+  <project name="seL4_tools.git" path="tools/seL4" remote="seL4" revision="2f15ba2dc6a08c30b74c722b0673131d2aab32ac" upstream="master" dest-branch="master">
```

Comment:
Similarly to previous comment, sel4devkit appears to have Jun28 commits subsequent to the foundation's 2f15ba2dc6a08c30b74c722b0673131d2aab32ac commit. I'd like to establish which commit is the closest equivalence or what the differences are. Can we do this?

Reply:
The comparison url for this example is https://github.com/sel4devkit/seL4_tools/compare/master..seL4:seL4_tools:2f15ba2dc6a08c30b74c722b0673131d2aab32ac again where green pluses signify lines that are in the Foundation repo, not our local one, with this ordering.

Reply:
Note: we walked through this diff in our team meeting and were satisfied that the differences were not significant. However, to confirm the first diff (for cmake-tool/helpers/application_settings.cmake) I will ensure a fresh test build of 32-bit seL4Test passes - TBD.

Reply:
> Note: we walked through this diff in our team meeting and were satisfied that the differences were not significant. However, to confirm the first diff (for cmake-tool/helpers/application_settings.cmake) I will ensure a fresh test build of 32-bit seL4Test passes - TBD.

Realised that AArch32 seL4Test is not actually going to be relevant here. seL4Test uses its own sel4test-manifest, not camkes-manifest which is being reviewed here. We use the seL4 Foundation's sel4test-manifest, since seL4Test supports MaaXBoard "natively" since our Foundation PRs; seL4/sel4test-manifest uses seL4/seL4_tools at a later revision than 2f15ba2 (9eef25a currently) and has already been tested.